oshc/main/static/main/js/script.js

```diff
@@ -5,4 +5,15 @@ $(document).ready(function() {
       scrollTop: 0
     }, 1000);
   });
+
+  $(".dropdown").hover(
```

Comment:
I have added just like if user hovers session, dropdown menu automatically popups. If you want i can remove